### PR TITLE
Ref [TASK][53248] Create UI home screen

### PR DESCRIPTION
**Pull request by ndt0409**
## Related Tickets
- [#Task 53248: Create UI home screen](https://edu-redmine.sun-asterisk.vn/issues/53248)
## WHAT
- 
## Evidence (Screenshot or Video)
image

## Review Checklist

Category | View Point | Description | Self review | Reviewer2 (name)
--- | --- | --- | --- | ---
Conventions | Sun* coding conventions | https://sal.vn/WtYPAy |<li>- [ ] yes</li>|<li>- [ ] yes</li>  
Conventions | Kotlin coding conventions | https://kotlinlang.org/docs/coding-conventions.html |<li>- [ ] yes</li>|<li>- [ ] yes</li>  
Redmine | Sun* Redmine working process  | https://sal.vn/piPoQn |<li>- [ ] yes</li>|<li>- [ ] yes</li>  

## Notes (Optional)
*(Impacted Areas in Application(List features, api, models or services that this PR will affect))*

*(List gem, library third party add new)*

*(Other notes)*

### CODE REVIEW

File: app/src/main/res/anim/exit_left_to_right.xml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<set xmlns:android="http://schemas.android.com/apk/res/android">
+    <translate
+        android:duration="500"
+        android:fromXDelta="0%"
+        android:fromYDelta="0%"
+        android:toXDelta="100%"
+        android:toYDelta="0%"/>
+</set>
```
Comment (minhnn-1279) [inline app/src/main/res/anim/exit_left_to_right.xml]: endline nhé a

File: app/src/main/res/anim/exit_right_to_left.xml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+<set xmlns:android="http://schemas.android.com/apk/res/android">
+    <translate
+        android:duration="500"
+        android:fromXDelta="0%"
+        android:fromYDelta="0%"
+        android:toXDelta="-100%"
+        android:toYDelta="0%"/>
+</set>
```
Comment (minhnn-1279) [inline app/src/main/res/anim/exit_right_to_left.xml]: đây cx endline nhé a

File: app/src/main/res/layout/toolbar.xml [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+                android:layout_height="@dimen/dp_50"
+                android:adjustViewBounds="true"
+                android:background="@drawable/logo"
+                android:scaleType="centerCrop"
+                android:visibility="visible" />
+
+            <TextView
+                android:id="@+id/subtitle_toolbar"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
```
Comment (ghost) [inline app/src/main/res/layout/toolbar.xml:30]: anh đặt id theo hướng dẫn này nhá: https://github.com/framgia/coding-standards/blob/master/vn/android/coding_convention_android_kotlin.md#32-resources-naming

File: app/src/main/java/com/ndt/themoviedb/ui/navigation/ContainerFragment.kt [hunk 4]
```diff
@@ -1,11 +1,55 @@
...
+    }
+
+    private fun loadFragment(fragment: Fragment, position: Int): Boolean {
+        when {
+            startingPosition > position -> {
+                activity?.supportFragmentManager?.beginTransaction()?.setCustomAnimations(
+                    R.anim.left_to_right,
+                    R.anim.exit_left_to_right,
+                    R.anim.right_to_left,
+                    R.anim.exit_right_to_left
```
Comment (vietnh-1109) [inline app/src/main/java/com/ndt/themoviedb/ui/navigation/ContainerFragment.kt]: cái naỳ ông viết riêng ra thành 1 object animation nhé , ví dụ slideLeftToRight , sliderightToLeft

File: app/src/main/res/layout/fragment_home.xml [hunk 5]
```diff
@@ -0,0 +1,163 @@
...
+        app:layout_constraintTop_toTopOf="parent">
+
+        <ProgressBar
+            android:id="@+id/home_progress_bar"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_gravity="center"
+            android:indeterminateTint="@color/color_orange" />
+    </FrameLayout>
+</androidx.constraintlayout.widget.ConstraintLayout>
```
Comment (vietnh-1109) [inline app/src/main/res/layout/fragment_home.xml]: endline

File: app/src/main/res/values/dimens.xml [hunk 6]
```diff
@@ -0,0 +1,14 @@
...
+    <dimen name="dp_0">0dp</dimen>
+    <dimen name="dp_275">275dp</dimen>
+    <dimen name="dp_16">16dp</dimen>
+    <dimen name="dp_8">8dp</dimen>
+    <dimen name="dp_5">5dp</dimen>
+    <dimen name="dp_18sp">18sp</dimen>
+    <dimen name="dp_24">24dp</dimen>
+    <dimen name="dp_50">50dp</dimen>
+    <dimen name="height_view_pager">275dp</dimen>
+</resources>
```
Comment (vietnh-1109) [inline app/src/main/res/values/dimens.xml]: endline

File: app/src/main/res/layout/fragment_home.xml [hunk 7]
```diff
@@ -0,0 +1,163 @@
...
+                    app:layout_constraintEnd_toEndOf="parent"
+                    app:layout_constraintHorizontal_bias="0.0"
+                    app:layout_constraintStart_toStartOf="parent"
+                    app:layout_constraintTop_toBottomOf="@+id/indicatorTabLayout" />
+
+                <com.google.android.material.tabs.TabLayout
+                    android:id="@+id/movie_by_key_tab_layout"
+                    android:layout_width="match_parent"
+                    android:layout_height="wrap_content"
+                    android:background="@android:color/black"
```
Comment (vietnh-1109) [inline app/src/main/res/layout/fragment_home.xml]: dùng màu black của resource color.xml ấy ông

**Review by minhnn-1279 — APPROVED** [review]

**Pull request merged**